foxy/distribution.yaml

```diff
@@ -596,6 +596,38 @@ repositories:
       url: https://github.com/CLOBOT-Co-Ltd/clober_msgs.git
       version: foxy-devel
     status: developed
+  clpe_ros:
+    doc:
+      type: git
+      url: https://github.com/canlab-co/clpe_ros.git
+      version: 0.1.0
+    release:
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/canlab-co/clpe_ros-release.git
+      version: 0.1.0
+    source:
+      test_pull_requests: true
+      type: git
+      url: https://github.com/canlab-co/clpe_ros.git
+      version: main
+    status: maintained
+  clpe_ros_msgs:
+    doc:
+      type: git
+      url: https://github.com/canlab-co/clpe_ros_msgs.git
+      version: 0.1.0
+    release:
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/canlab-co/clpe_ros_msgs-release.git
+      version: 0.1.0
```

Comment:
There are [no releases for this package at the provided URL](https://github.com/canlab-co/clpe_ros_msgs-release/branches). Same for `clpe_ros-release`. If you want to just index the packages, you can remove the `release` entries from this pull request, leaving the `doc` and `source` entries.